### PR TITLE
feat: `bloomWallet`

### PR DESCRIPTION
**Pull request by 0xmkh**
## Changes
- Added `bloomWallet`

## What to test
1. Install Bloom Wallet on desktop [here](https://bloomwallet.io/)
2. Once installed open the desktop app and create a wallet
3. Go to RainbowKit modal and make sure the wallet connection works smoothly

### CODE REVIEW

**Comment by KosmosKey** [comment]
Bloom wallet seems to have a small bug during on connection:

Screenshot 2024-02-20 at 22 54 56

File: site/data/en-US/docs/custom-wallet-list.mdx [hunk 1]
```diff
@@ -142,6 +142,14 @@ import { bitskiWallet } from '@rainbow-me/rainbowkit/wallets';
 bitskiWallet();
 ```
```
Comment (DanielSinclair) [inline site/data/en-US/docs/custom-wallet-list.mdx:144]: Looks like we're missing a title here

File: packages/rainbowkit/src/wallets/walletConnectors/index.ts [hunk 2]
```diff
@@ -82,4 +83,5 @@ export {
   xdefiWallet,
   zealWallet,
   zerionWallet,
+  bloomWallet,
```
Comment (DanielSinclair) [inline packages/rainbowkit/src/wallets/walletConnectors/index.ts:86]: I'd alphabetize this for easier maintenance

File: packages/example/pages/_app.tsx [hunk 3]
```diff
@@ -175,6 +176,7 @@ const config = getDefaultConfig({
         xdefiWallet,
         zealWallet,
         zerionWallet,
+        bloomWallet,
```
Comment (DanielSinclair) [inline packages/example/pages/_app.tsx:179]: Would alphabetize here too

File: .changeset/silly-dolls-grow.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+"@rainbow-me/rainbowkit": patch
+"example": patch
+"site": patch
+---
+
+Added `bloomWallet`
```
Comment (DanielSinclair) [inline .changeset/silly-dolls-grow.md:7]: Might change to something like this with a code sample.

Bloom Wallet support

```tsx
import { bloomWallet } from '@rainbow-me/rainbowkit/wallets';
const config = getDefaultConfig({
  wallets: [bloomWallet],
});
```

**Review by DanielSinclair — APPROVED** [review]
Approved for when you're ready with the small changes

**Comment by MarkNerdi** [comment]
> Bloom wallet seems to have a small bug during on connection:
> 
> Screenshot 2024-02-20 at 22 54 56

The "problem" here is that currently Bloom only supports eip155:148 and eip155:1074. We already worked on improving the UX by showing explicitly what went wrong.

**Review by DanielSinclair — APPROVED** [review]

**Pull request merged**